com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointContinuous.cs

```diff
@@ -115,6 +116,13 @@ protected override Joint ExportSpecificJointData(Joint joint)
 #endif
             return joint;
         }
+        
+        protected Joint.Axis GetAxisData()
+        {
+            var res = (-1 * (unityJoint.anchorRotation * new Vector3(1, 0, 0)));//.Unity2Ros();
```

Comment:
Unity2Ros() - looks like it needs to be uncommented to convert to LHS that's used in Unity.